### PR TITLE
Add cypress test

### PR DESCRIPTION
**Pull request by Jin-Sun-tts**
# Pull Request

related to https://github.com/GSA/data.gov/issues/3521

## About

Add Cypress tests (or other UI tests) implemented in datagovtheme.

## PR TASKS

- [ ] ~The actual code changes.~
- [x] Tests written and passed.
- [x] Added new directory e2e for Cypress tests
- [x] Bumped version number in [setup.py](https://github.com/GSA/ckanext-geodatagov/blob/1b1bad0b2ff06112e18c7f4f4fb1143baec1266a/setup.py#L13) (also checked on [PyPi](https://pypi.org/project/ckanext-geodatagov/#history)).

### CODE REVIEW

File: e2e/cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (jbrown-xentity) [inline e2e/cypress/fixtures/example.json:5]: Was this just a test implementation? Do we need this?

File: e2e/cypress/support/commands.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+//       url: '/api/3/action/package_create',
+//       headers: {
+//           'cache-control': 'no-cache',
+//           'content-type': 'application/json',
+//       },
+//       body: JSON.stringify(ckan_dataset),
+//   };
+
+//   return cy.request(options);
+// });
```
Comment (jbrown-xentity) [inline e2e/cypress/support/commands.js:169]: Since we're no longer using harvest sources to test, should we remove all this code?

**Review by jbrown-xentity — APPROVED** [review]
In general looks good! I would like to somehow normalize `ui-test` with `test`, as we're definitely doing similar things in 2 very different ways. That can be done in a follow up however.

File: e2e/cypress/fixtures/example.json (hunk 1 above)
Comment (Jin-Sun-tts) [reply e2e/cypress/fixtures/example.json]: good catch, we do not need it. just removed it.

File: e2e/cypress/support/commands.js (hunk 2 above)
Comment (Jin-Sun-tts) [reply e2e/cypress/support/commands.js]: yes, just removed those unused files.

**Review by nickumia-reisys — APPROVED** [review]
🌴

**Pull request merged**